### DISCUSSION TITLE
Would it be possible for you to release the 5.4 WSL kernel as a beta alongside the current stable version?

### DISCUSSION THREAD

**Discussion by gut5** [microsoft/WSL#6026] [General] [answered]
no text

**Comment by 666threesixes666** [comment]
i second this!  my kernels old as hell, what gives?

**Comment by Biswa96** [comment]
You could download the Linux kernel upadate from here https://www.catalog.update.microsoft.com/Search.aspx?q=wsl

**Comment by craigloewen-msft** [comment]
Where would you want to see this kernel be available to download from? A link to the MSI on the release notes? 

In general, for kernel servicing we push a new version to Insiders and then publish it to all WSL builds after verification in the Insiders program. As of today we moved the 5.4 kernel to be available on the Beta and Preview Release channel as well.

**Reply by gut5** [reply]
"A link to the MSI on the release notes?" 
Yes, this would be okay thanks

**Comment by craigloewen-msft** [comment, marked as answer]
I've found a solution here. You can download the latest kernel updates from the Microsoft Update Catalogue if you'd like to gain early access. 

https://www.catalog.update.microsoft.com/Search.aspx?q=wsl
